### ISSUE TITLE
20131019_Petrovtsi_Bitva_Za_Kiev/ by the tag "Військовий об'єкт" in the search, the trip "Битва за Київ у 1943 році" is not found

### ISSUE THREAD

**Issue by Illia2810** [scholokov/long-travel-2#4912]
Pre-conditions:
Open: https://test.long-travel.live/travels/

Steps to reproduce:

Note that by the tag "Військовий об'єкт" in the search, the trip "Битва за Київ у 1943 році" is not found

Actual result:
by the tag "Військовий об'єкт" in the search, the trip "Битва за Київ у 1943 році" is not found
Expected result:
according to search requirements for the tag "Військовий об'єкт" should find the trip "Битва за Київ у 1943 році"

Environment:
System: Windows 10 Pro
Browser: FireFox

Requirements:
https://qax.gitbook.io/qax-travel/requirements/4.travel/travels-list-1 - 012

Attachment:
![зображення](https://github.com/scholokov/long-travel-2/assets/125801673/1ca13ee6-406e-4370-a399-619cfd35b6fc)

**Comment by Illia2810** [comment]
#4912

**Comment by DmytroMarkulych** [comment]
Можливо в подорожі немає тегів

**Comment by DmytroMarkulych** [comment]
![Image](https://github.com/scholokov/long-travel-2/assets/88647733/72ed134b-7c5f-4175-9168-f5cd3e5952be)

**Comment by Illia2810** [comment]
так, не має, але основне тут, ми відштовхуючтсь від фльтра, а за фільтром "Військовий об'єкт" в самому пошуку, цю подорож не знаходити

**Comment by DmytroMarkulych** [comment]
ну так, бо відсутня умова по якій шукати

**Comment by Illia2810** [comment]
виправлено

**Comment by scholokov** [comment]
fixed

**Comment by Illia2810** [comment]
verified
